### PR TITLE
04 get /api/reviews

### PR DESCRIPTION
**Pull request by jake4369**
Created model, controller, route and test for 4. GET /api/reviews

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -11,21 +11,56 @@ beforeEach(() => {
...
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("review_id", expect.any(Number));
```
Comment (HannahBurks) [inline __tests__/app.test.js:46]: We seem to be checking for review_id twice here, but what we really want to check for is comment_count :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -11,21 +11,56 @@ beforeEach(() => {
...
+        });
+      });
+  });
+  it("should respond with data sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        const sortedReviews = [...reviews].sort((a, b) =>
```
Comment (HannahBurks) [inline __tests__/app.test.js:56]: Here we can use jest.sorted to check this as opposed to manually sorting.

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Nice work Jake 👍🏼 just a few minor changes, mainly I think we have a typo in the test (we have checked for review_id twice and not comment_count) - it could also be a nice to have a test that checks the comment_count is correct for one specific object (to ensure the SQL is correct) Please drop another PR when done 😄

**Review by Geo-Beard — COMMENTED** [review]
Looks OK overall - would suggest using jest sorted instead of manually sorting and checking that. Not sure if the changes have come through properly - so if you've implemented that already no worries. Otherwise, implement jest sorted in your tests and resubmit tomorrow.

**Pull request merged**